elements/pl-matching/pl-matching.py

```diff
@@ -149,25 +150,35 @@ def prepare(element_html, data):
         more_needed = number_options - len(needed_options)
         if more_needed >= len(distractors):
             # Add all distractors.
```

Comment:
The original comment made more sense to me with the original implementation; hopefully this updated comment is more clear?
```suggestion
            # Use all options, both correct and distractors.
```